spec/controllers/cart_controller_spec.rb

```diff
@@ -1,86 +1,103 @@
 require 'spec_helper'
```

Comment:
These tests were copied from OrderController. The existing CartControllerSpec file was moved to where it should be under spec/controllers/open_food_network